src/flisp/Makefile

```diff
@@ -117,7 +117,7 @@ $(BUILDDIR)/host/Makefile:
 	@printf "%s\n" 'include $(SRCDIR)/Makefile' >> $@
 
 $(BUILDDIR)/host/$(EXENAME): $(BUILDDIR)/host/Makefile | ${BUILDDIR}/host/flisp.boot
-	$(MAKE) -C $(BUILDDIR)/host $(EXENAME)
+	make -C $(BUILDDIR)/host $(EXENAME)
```

Comment:
This actually should be `$(MAKE)`; if this works as-is on FreeBSD then it must not be getting called there since `make` is not GNU Make.